types/react-native/index.d.ts

```diff
@@ -3998,12 +3999,17 @@ export interface ViewabilityConfig {
     waitForInteraction?: boolean | undefined;
 }
 
-export interface ViewabilityConfigCallbackPair {
+export interface ViewabilityConfigCallbackPair<ItemT, SectionT = never> {
```

Comment:
Why does `ItemT` not have a default?

Reply:
Good point - on second thoughts it probably should, let me fix that up

Reply:
These are added